ml/src/main/scala/ai/h2o/sparkling/ml/features/H2OTargetEncoder.scala

```diff
@@ -74,6 +74,8 @@ class H2OTargetEncoder(override val uid: String)
 
   def setInputCols(values: Array[String]): this.type = set(inputCols, values)
 
+  def setOutputCols(values: Array[String]): this.type = set(outputCols, values)
```

Comment:
Should we check that `values` can't be null?

Reply:
That's a really good point! In Scala world, it's not a practice, but an user can pass anything. Do check this in other parts of API? I don't have any strong opinion about this, I will tailor it according to you preferences ;-)

Reply:
After today's discussion I would stick with the current API :)